librz/include/rz_bin.h

```diff
@@ -161,6 +161,8 @@ enum {
 	RZ_STRING_TYPE_UTF8 = 'u',
 	RZ_STRING_TYPE_WIDE = 'w', // utf16 / widechar string
```

Comment:
While we are on it, I wonder if we should make it clear that those `RZ_STRING_TYPE_WIDE*` are in fact `WIDE*_LE`. cc @ret2libc @kazarmy @wargio @pelijah

Reply:
better to rename it